insights-notification-schemas-java/src/test/java/com/redhat/cloud/notifications/ingress/TestSerialization.java

```diff
@@ -132,13 +133,30 @@ void shouldHaveDefaultValuesWhenNotSet() {
      }
 
      @Test
-     void shouldAcceptExtraPropertiesAtEventLevel() {
-         Action action = getValidAction();
-         action.getEvents().get(0).setAdditionalProperty("report_url", "http://blablabla.com");
+     void shouldAcceptExtraPropertiesAtEventLevelWhenDeserializingAndFailIfSerializingOrValidating() {
```

Comment:
`throw new TooLongDidntReadException`

I'm joking, we can keep it that way :smiley: